doc/conf.py

```diff
@@ -88,6 +88,13 @@ def _check_deps():
 
 autodoc_docstring_signature = True
 
+intersphinx_mapping = {
+  'python': ('http://docs.python.org/', None),
+  'numpy': ('http://docs.scipy.org/doc/numpy-dev/', None),
```

Comment:
We should link to the stable version of numpy docs (https://docs.scipy.org/doc/numpy/)